ext/nginx.conf

```diff
@@ -25,6 +36,9 @@ server {
 	location / {
 		proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
 		proxy_set_header X-Real-IP  $remote_addr;
+		proxy_set_header X-Country-Code $geoip2_data_country_code;
+		proxy_set_header X-Latitude $geoip2_data_latitude;
+		proxy_set_header X-Longitude $geoip2_data_longitude;
```

Comment:
should this be commented out by default? (will this throw if the module is not enabled?)

Reply:
would bpy not crap out either way if it wasn't enabled?

Reply:
it has an api-based backup approach if it is disabled. (we need to ensure that's actually working)

Reply:
see https://github.com/osuAkatsuki/bancho.py/pull/367/files#diff-4534a559c52f8fe8ea43b1b8d0649adef038fc8530855d8c7234c7ad5372cac8R1954

Reply:
also i have a suggestion for this then:

1. set default value for those variables
2. check if default value is present on those headers in fetch_nginx to see if maxminddb enabled

Reply:
```nginx
set $geoip2_data_country_code 'default';
...
```

Reply:
Default values are here https://github.com/osuAkatsuki/bancho.py/pull/367/files#diff-74e6757613db9ff8909a81b274ee6a836e1429d23734a01820bd0a00194315afR18-R20
I agree with the second point

Reply:
oh, hadn't noticed defaults were set

Reply:
> should this be commented out by default? (will this throw if the module is not enabled?)

if the module isn't loaded or installed nginx will throw this error
![image](https://user-images.githubusercontent.com/33896771/212441642-f8660533-f48b-4649-87ed-ffe6fdaab4ec.png)